.github/dependabot.yml

```diff
@@ -4,6 +4,10 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+      # run these when most of our developers don't work, don't DoS our CI over the day
+      time: "22:00"
```

Comment:
Should we differentiate Cockpit / machines times. So say machines starts at 22:30, podman 23:00?